Assets/Scripts/CoinSpawner.cs

```diff
@@ -79,10 +79,15 @@ private IEnumerator RespawnTimer()
             if (SecondsToRespawn >= 0)
             {
                 yield return new WaitForSeconds(SecondsToRespawn);
-                mPhotonView.RPC("Respawn", PhotonTargets.All);
+                RespawnOverNetwork();
             }
         }
 
+        public void RespawnOverNetwork()
+        {
+            mPhotonView.RPC("Respawn", PhotonTargets.All);

```

Comment:
Oooo, actually I think you are right this should be buffered because coin collection/despawn is buffered. I'm on the road headed back to town but I won't be there until later tonight. I'll double check once I get back whether buffered is teally needed or not and I'll retest if it turns out we need it to be buffered (which I think is true for games of 3+ players which I did not test)